docs/src/pages/getting-started/templates/checkout/Checkout.tsx

```diff
@@ -151,3 +154,11 @@ export default function Checkout() {
     </React.Fragment>
   );
 }
+
+export default function Checkout() {
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
We might want to leave a TODO note so we can easily remove them in the future.

Reply:
Done